tools/sdk-automation-core/LICENSE

```diff
@@ -0,0 +1,21 @@
+    MIT License
```

Comment:
This should be removed as we have it in the license.

Reply:
Removed.